src/components/DescriptionList/DescriptionList.scss

```diff
@@ -39,6 +41,8 @@ $breakpoint: 550px;
   @include page-content-breakpoint-after($breakpoint) {
     flex: 1 1 51%;
     padding: spacing() 0;
+    overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
Should we move all changes outside the media query?

The list is also not rendering properly on mobile:
![Mar-31-2020 10-39-41](https://user-images.githubusercontent.com/3925905/78039528-399c5480-733c-11ea-8115-c19eef7d80c7.gif)


With the fix:
<img width="1116" alt="Screen Shot 2020-03-31 at 10 38 56 AM" src="https://user-images.githubusercontent.com/3925905/78039559-415bf900-733c-11ea-825c-c2a02be07aa7.png">

Reply:
Good point 🤔 

Although it looks strange to have to side scroll to see the content, adding an ellipsis for mobile now makes that content unreadable after it overflows.

I've updated the PR to break the words for mobile, so that it doesn't give a scrollable x-axis and instead stacks the content.

| ![resizeDescription](https://user-images.githubusercontent.com/5506721/78053216-46c23f00-734e-11ea-9e66-c4414b64f509.gif)|
|---|

wdyt @loic-d ?  _it may depend on some UX feedback as well for this decision_

Reply:
Looks great! Wondering if we should use the word break for desktop as well then? Let's see what UX says...